packages/cardano-services/src/Asset/AssetBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import { Cardano } from '@cardano-sdk/core';
+import { LastMintTxModel, MultiAssetHistoryModel, MultiAssetModel, MultiAssetQuantitiesModel } from './types';
+import { Logger, dummyLogger } from 'ts-log';
+import { Pool, QueryResult } from 'pg';
+import Queries from './queries';
+
+export class AssetBuilder {
+  #db: Pool;
+  #logger: Logger;
+
+  constructor(db: Pool, logger = dummyLogger) {
```

Comment:
Default to dummyLogger can lead to silently swallowing the error. Let's try to avoid it and only use dummyLogger in tests.

```suggestion
  constructor(db: Pool, logger: Logger) {
```

Reply:
Resolved in 6cb147271

Reply:
This is still unresolved @iccicci. The `fixup!` hasn't been squashed, and is not disconnected from the branch

![image](https://user-images.githubusercontent.com/303881/179668023-429538a0-4670-40e7-b3a6-d85401e8a5c6.png)